adminSiteClient/GrapherConfigGridEditor.tsx

```diff
@@ -91,6 +94,57 @@ import {
 import { Query, Utils as QbUtils } from "react-awesome-query-builder"
 // types
 import { SimpleField, Config, ImmutableTree } from "react-awesome-query-builder"
+import { KeysSection } from "./EditorDataTab.js"
+import codemirror from "codemirror"
+import { UnControlled as CodeMirror } from "react-codemirror2"
+import jsonpointer from "json8-pointer"
+import { EditorColorScaleSection } from "./EditorColorScaleSection.js"
+import { MapChart } from "../grapher/mapCharts/MapChart.js"
+
+function HotColorScaleRenderer(props: Record<string, unknown>) {
+    return <div>Color scale</div>
+}
+
+class HotColorScaleEditor extends BaseEditorComponent<Record<string, never>> {
```

Comment:
you never really use this component.
I guess it was an experiment to show the color editor inline, which didn't work out?
should probably get rid of it then?

Reply:
I clarified in a comment why this is needed:  This cell editor component explicitly does nothing. It is used for color scale columns where we don't want to make the cells read only so that copy/paste etc still work but at the same time we don't want the user to be able to edit the cells directly (instead they should use the sidebar editor panel)